src/Components/TabletView/QuickBook.tsx

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import {
+    Box,
+    Button,
+    Typography,
+    Grid,
+    Paper,
+    TextField,
+    Autocomplete,
+} from "@mui/material";
+import EditCalendarIcon from "@mui/icons-material/EditCalendar";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { styled } from "@mui/material/styles";
+
+const Item = styled(Paper)(({ theme }) => ({
+    backgroundColor: theme.palette.mode === "dark" ? "#1A2027" : "#fff",
+    ...theme.typography.body2,
+    padding: theme.spacing(1),
+    textAlign: "center",
+    color: theme.palette.text.secondary,
+}));
+const buttonStyle = { color: "#008435", border: "1px solid #008435" };
+
+const QuickBook = (props: any) => {
+    const [bookingIsOpen, setBookingIsOpen] = useState(false);
+    const [flag, setFlag] = useState(false);
+    const [open, setOpen] = React.useState(false);
```

Comment:
Be consistent with useState, don't use React.useState

Reply:
Fixed that